lib/pf/UnifiedApi/Controller/Services.pm

```diff
@@ -0,0 +1,113 @@
+package pf::UnifiedApi::Controller::Services;
+
+=head1 NAME
+
+pf::UnifiedApi::Controller::Services -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::UnifiedApi::Controller::Services
+
+=cut
+
+use strict;
+use warnings;
+use Mojo::Base 'pf::UnifiedApi::Controller::Crud';
+use pf::services;
+
+
+sub status {
+    my ($self) = @_;
+    my $class = "pf::services::manager::".$self->param('service');
```

Comment:
Create a helper method in this module that handles fetching the class and rendering the error if necessary instead of having the same code in all the subroutines

Reply:
This here should also handle the expansion of the service name (the API should accept httpd.admin as the service parameter, not httpd_admin since the name of the service is httpd.admin)

You could probably loop in the services manager list and search by the name (@pf::services::MANAGERS)